data/templates/bootstrap.js

```diff
@@ -54,9 +53,13 @@ function eachSeries(arr, iterator, callback) {
  * @private
  * @param data
  * @param timeout
- * @param retries
  */
-function updateTasks(data, timeout, retries) {
+function updateTasks(data, timeout) {
+    // value to check succes case.
```

Comment:
These comments are redundant, it's pretty clear from the code what is happening already.